server/data/bnf/helpers.js

```diff
@@ -0,0 +1,26 @@
+import { compact, uniq } from 'lodash-es'
+import { resolveExternalIds } from '#controllers/entities/lib/resolver/resolve_external_ids'
+import { isWdEntityUri } from '#lib/boolean_validations'
+
+export async function setEditionContributors (edition, property, urls) {
+  const uris = await resolveContributors(urls)
+  if (uris.length > 0) {
+    edition.claims[property] = uris
```

Comment:
There is a common pattern of seeds sometimes assigning translators as P50. But  this will not conflict with the possibility of already having those authors in the P50 claim, right? aka BNF based entity creation is not based on the seed, right?